### ISSUE TITLE
Thor: Warn text on all commands "expected string default value for '--force'; got false (boolean)

### ISSUE THREAD

**Issue by shadowbq** [technicalpickles/homesick#156]
```
$> homesick
Expected string default value for '--force'; got false (boolean)
Expected string default value for '--force'; got false (boolean)
Commands:
  homesick cd CASTLE              # Open a new shell in the root of the given castle
  homesick clone URI CASTLE_NAME  # Clone +uri+ as a castle with name CASTLE_NAME for homesick
[..snip..]
  homesick status CASTLE          # Shows the git status of a castle
  homesick track FILE CASTLE      # add a file to a castle
  homesick unlink CASTLE          # Unsymlinks all dotfiles from the specified castle
  homesick version                # Display the current version of homesick

Runtime options:
  -f, [--force]                    # Overwrite files that already exist
  -p, [--pretend], [--no-pretend]  # Run but do not make any changes
  -q, [--quiet], [--no-quiet]      # Suppress status output
  -s, [--skip], [--no-skip]        # Skip files that already exist
```

thor (0.19.4)

```
   # TODO: This should raise an ArgumentError in a future version of Thor
      warn "Expected #{@type} default value for '#{switch_name}'; got #{@default.inspect} (#{default_type})" unless default_type == @type
```

bash
ruby 2.2.6p396 (2016-11-15 revision 56800) [amd64-freebsd11]
homesick (1.1.3)

**Comment by shadowbq** [comment]
https://github.com/erikhuda/thor/blob/1a8b1b397804895df5973afd12cf2823855457ca/lib/thor/parser/option.rb

**Comment by shadowbq** [comment]
this looks like it was fixed in master, but the fix has not been released as a gem.

**Comment by stratosgear** [comment]
Hi, yes this is pretty annoying!

The first thing I saw after installation was this error message. Not a very good impression.

Especially when the fix is already, implemented in master.  Please make a new release!

**Comment by JCook21** [comment]
I'm going to cut a new release in the next couple of days, as soon as I have time to do it. Thanks for your patience.

**Comment by JCook21** [comment]
I've just released version 1.1.4. Can you verify that this is fixed there @stratosgear and @shadowbq  and close this issue if so?

**Comment by JCook21** [comment]
@JoeNyland thanks for verifying this. Could you create a new issue for this? It looks like something went wrong in between bumping the version, merging to master and cutting the new gem *facepalm*. I'll have to cut a new version to resolve this later.

In the meantime I'll close this issue. Feel free to reopen if you think there's a need.
